plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -741,6 +741,9 @@ public String getColumnTypeModifiers(@NotNull DBPDataSource dataSource, @NotNull
                         return null;
                     }
                     return "(" + maxLength + ")";
+                } if ("VARCHAR".equals(typeName)) {
+                    // Default max length value for varchar column, because many databases do not support varchar without modifiers
+                    return "(100)";
```

Comment:
Maybe but it higher so the `FEATURE_MAX_STRING_LENGTH` feature is respected?

Reply:
I checked. This max values look to scary. 65535 for MySQL. 10485760 for PostgreSQL. Not sure.